File: exercise/payrate(test2).c
#include <stdio.h>

int payRate;
int hoursWorked;
int grossPercent;
int grossPay;
int witholdingAmount;
int netPay;

int main(){

    printf("What is the company pay rate: ");
    scanf("%d", &payRate);

    printf("How many hours did in worker worked: ");
    scanf("%d", &hoursWorked);

    printf("What is the gross salary percentage: ");
    scanf("%d", &grossPercent);

    grossPay = payRate*hoursWorked;
    witholdingAmount = grossPay*(grossPercent/100);
    netPay = grossPay-witholdingAmount;

    printf("\nThe net pay is: %d", netPay);  
}

File: exercise/payrate(test2).py
payRate = int(input("What is the company pay rate: "))

hoursWorked = int(input("How many hours did in worker worked: "))

grossPercent = int(input("What is the gross salary percentage: "))

grossPay = payRate*hoursWorked

witholdingAmount = grossPay*(grossPercent/100)

netPay = grossPay-witholdingAmount

print ("The net pay is: ", netPay)